packaging/post-install.sh

```diff
@@ -188,6 +188,17 @@ function get_extension_file() {
     echo "${EXTENSION_DIR}/elastic_apm-${PHP_API}${SUFFIX}.so"
 }
 
+################################################################################
+#### Function is_php_supported #################################################
+function is_php_supported() {
+    PHP_MAJOR_MINOR=$(php_command -r 'echo PHP_MAJOR_VERSION;').$(php_command -r 'echo PHP_MINOR_VERSION;')
+    if  [ "${PHP_MAJOR_MINOR}" == "7.2" ] || [ "${PHP_MAJOR_MINOR}" == "7.3" ] || [ "${PHP_MAJOR_MINOR}" == "7.4" ] ; then
+        return 0
+    else
+        return 1
+    fi
```

Comment:
@mdelapenya , what do you think this suggestions instead https://github.com/elastic/apm-agent-php/pull/330/files#r560268450?

```suggestion
    PHP_MAJOR_MINOR=$(php_command -r 'echo PHP_MAJOR_VERSION;').$(php_command -r 'echo PHP_MINOR_VERSION;')
    echo "Detected PHP version '${PHP_MAJOR_MINOR}'"
    if  [ "${PHP_MAJOR_MINOR}" == "7.2" ] || [ "${PHP_MAJOR_MINOR}" == "7.3" ] || [ "${PHP_MAJOR_MINOR}" == "7.4" ] ; then
        return 0
    else
        echo 'Failed. The supported PHP versions are 7.2, 7.3 and 7.4.'
        return 1
    fi
```

Reply:
Sounds great!